src/app/pages/ArticlePage/experimentTopStories/helpers.test.tsx

```diff
@@ -16,11 +16,13 @@ describe('AMP top stories experiment', () => {
       blocks: [],
     },
   };
-  const expectedExperimentTopStoriesBlock = (index: number) => {
+  const expectedExperimentTopStoriesBlock = (
+    variant: 'Quarter' | 'Half' | 'ThreeQuarters',
+  ) => {
     return {
-      type: 'experimentTopStories',
+      type: `experimentTopStories${variant}`,
       model: topStoriesList,
-      id: `experimentTopStories-${index}`,
+      id: `experimentTopStories${variant}`,
```

Comment:
There will only ever be one of each variant in the blocks array, so a numeric index is no longer needed.